layouts/shortcodes/feature-state.html

```diff
@@ -1,10 +1,11 @@
 {{ $valid_states := "alpha, beta, deprecated, stable" }}
 {{ $state := .Get "state" }}
-{{ $for_k8s_version := .Get "for_k8s_version" }}
-{{ $width := .Get "width" }}
+{{ $for_k8s_version := .Get "for_k8s_version" | default (.Page.Param "version")}}
 {{ $is_valid := strings.Contains $valid_states $state }}
 {{ if not $is_valid }}
 {{ errorf "%q is not a valid feature-state, use one of %q" $valid_states }}
 {{ else }}
-{{ partial "templates/feature-state.html" (dict "ctx" . "page" .Page "state" $state "for_k8s_version" $for_k8s_version "width"  $width) }}
-{{ end }}
+<div style="margin-top: 10px; margin-bottom: 10px;">
+<b>FEATURE STATE:</b> <code>Kubernetes {{ $for_k8s_version }} [{{ $state }}]</code>
```

Comment:
@jimangel, Here is an _idea_. It would be interesting to _link_ the `state` to the state definition.

The definitions appear in a number of places in the docs, such as:
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages.
The info in that page could be updated to create H3 level headings for each state type so that the state can link to the correct heading on the page.

```
## Using a feature

A feature can be in *Alpha*, *Beta* or *GA* stage.

### Alpha features

* Disabled by default.
* Might be buggy. Enabling the feature may expose bugs.
* Support for feature may be dropped at any time without notice.
* The API may change in incompatible ways in a later software release without notice.
* Recommended for use only in short-lived testing clusters, due to increased
  risk of bugs and lack of long-term support.

### Beta features

* Enabled by default.
* The feature is well tested. Enabling the feature is considered safe.
* Support for the overall feature will not be dropped, though details may change.
* The schema and/or semantics of objects may change in incompatible ways in a
  subsequent beta or stable release. When this happens, we will provide instructions
  for migrating to the next version. This may require deleting, editing, and
  re-creating API objects. The editing process may require some thought.
  This may require downtime for applications that rely on the feature.
* Recommended for only non-business-critical uses because of potential for
  incompatible changes in subsequent releases. If you have multiple clusters
  that can be upgraded independently, you may be able to relax this restriction.

{{< note >}}
Please do try *Beta* features and give feedback on them!
After they exit beta, it may not be practical for us to make more changes.
{{< /note >}}

### Stable features

A *General Availability* (GA) feature is also referred to as a *stable* feature. It means:

* The feature is always enabled; you cannot disable it.
* The corresponding feature gate is no longer needed.
* Stable versions of features will appear in released software for many subsequent versions.
```

Reply:
I started to play with links but since it's dealing with raw HTML, I struggled to think how we could do this easily in a way that handles localization.

Do you have any thoughts on how to format the URL? If so, I would be open to adding that into this PR!

Reply:
At this moment, I do not have a quick solution.

Reply:
No worries! I think in that case, we merge this as is (if everyone agrees) and we can revisit in the future with further improvements. I like the idea though!